htdocs/bootstrap/css/custom-css.css

```diff
@@ -453,3 +453,28 @@ a.btn.btn-primary {
 .sweet-alert p {
   word-wrap: break-word;
 }
+
+/* Loader customizations */
+
+.loader-container {
+    height: 80vh;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.loader {
+    border: 8px solid #FFFFFF; /* White */
+    border-top: 8px solid #3498db; /* Blue */
+    border-radius: 50%;
+    width: 120px;
+    height: 120px;
```

Comment:
Think about what you'd have to do to get rid of the container for my comment above makes me think that maybe this should be something like:

```
width: 100%;
max-width: 120px;
```

But then you'd need some way to ensure height matches width or you'd get a weird oblong loader..